tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -154,6 +154,20 @@ kbs_set_resource_from_file() {
 	kbs-client --url "$(kbs_k8s_svc_http_addr)" config \
 		--auth-private-key "$KBS_PRIVATE_KEY" set-resource \
 		--path "$path" --resource-file "$file"
+
+	kbs_pod=$(kubectl -n coco-tenant get pods -o NAME)
```

Comment:
there should be `$KBS_NS`  instead of `coco-tenant` (here and also on line 163)

Reply:
Fixed, and also fixed in the loop where you missed that. Thanks!